README.md

```diff
@@ -6,70 +6,54 @@
 
 # Data is Better Together
 
-Data is Better Together is a collaboration between 🤗 Hugging Face, 🏓 Argilla, and the Open-Source ML community. We aim to empower the open-source community to build impactful datasets collectively. 
+Data is Better Together is a collaboration between 🤗 Hugging Face, 🏓 Argilla, and the Open-Source ML community. We aim to empower the open-source community to build impactful datasets collectively. This initiative consists of two main components: the community efforts and the cookbook efforts.
 
-## What have we done so far?
+<details open>
+  <summary><strong>Community Efforts</strong>: They were guided, hands-on projects focused on creating valuable datasets. These projects required the participation of the community and have been successfully completed.</summary>
```

Comment:
Maybe emphasize **GUIDED BY THE HF TEAM** and for the other ones **NOT GUIDED** and expected to be handled standalone.